src/config.ts

```diff
@@ -0,0 +1,47 @@
+import * as fs from 'fs'
+import YAML from 'yaml'
+import * as z from 'zod'
+import path from 'path'
+
+export type Severity = "critical" | "high" | "moderate" | "low"
+
+export const SEVERITIES = ["critical", "high", "moderate", "low"] as const
+export const CONFIG_FILEPATH = "./.github/dep-review.yml"
+
+type ConfigurationOptions = {
+    fail_on_severity: string,
+    allow_licenses: Array<string>,
+    deny_licenses: Array<string>
+}
+
+export function readConfigFile(filePath: string = CONFIG_FILEPATH): ConfigurationOptions {
+    // By default we want to fail on all severities and allow all licenses.
+    var defaultOptions: ConfigurationOptions = {
+        fail_on_severity: "low",
+        allow_licenses: ['all'],
+        deny_licenses: []
+    }
+
+    try {
+        var data = fs.readFileSync(path.resolve(filePath), "utf-8");
```

Comment:
Nit: for consistency and for avoiding surprises, I'd use `let` instead of `var` (`var` has different scoping rules).